### ISSUE TITLE
Add fields to the media object

### ISSUE THREAD

**Issue by g8rswimmer** [g8rswimmer/go-twitter#155]
Two fields where missing from the media object, alt text and variants.

https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/media
